developer-extension/src/panel/hooks/useEvents/eventCollection.ts

```diff
@@ -28,6 +26,17 @@ export function startEventCollection(strategy: EventCollectionStrategy, onEvents
   }
 }
 
+function compareEvents(a: SdkEvent, b: SdkEvent) {
+  // Sort events chronologically
+  if (a.date !== b.date) {
+    return b.date - a.date
+  }
+
+  // If two events have the same date, make sure to display View events last. This ensures that View
+  // updates are collocated in the list (no other event are present between two updates)
+  return (isRumViewEvent(a) as any) - (isRumViewEvent(b) as any)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** in which case could we have two view updates with the same date?

Reply:
We are talking about the event date here. So all View updates related to the same View have the same date.

Reply:
I'll add an example in the comment to make it clearer.